file_writer_control/WorkerJobPool.py

```diff
@@ -13,17 +13,25 @@ class WorkerJobPool(WorkerFinder):
     A child of WorkerFinder intended for use with "worker pool" style of starting a file-writing job.
     """
 
-    def __init__(self, job_topic_url: str, command_topic_url: str):
+    def __init__(
+        self,
+        job_topic_url: str,
+        command_topic_url: str,
+        max_message_size: int = 1048576 * 200,
+    ):
         """
         :param job_topic_url: The Kafka topic that the available file-writers are listening to for write jobs.
-        :param command_topic_url:  The Kafka topic that a file-writer uses to send status updates to and receive direct
+        :param command_topic_url: The Kafka topic that a file-writer uses to send status updates to and receive direct
+        :param max_message_size: The maximum message (actually "request") size.
```

Comment:
This line was inserted before the end of the previous parameter documentation (i.e. should be after what is currently line 26).